src/server/service/kv.rs

```diff
@@ -904,6 +878,7 @@ impl<T: RaftStoreRouter + 'static> tikvpb_grpc::Tikv for Service<T> {
 
         let future = sink.send_all(stream)
             .map(|_| timer.observe_duration())
+            .map_err(Error::from)
```

Comment:
Why?

Reply:
There is a `Sink` error in server errors so that I think it would be better to map the error to Sink Error before printing.

Reply:
It doesn't matter. `print` is only for debugging.